### DISCUSSION TITLE
Zos copy - fails

### DISCUSSION THREAD

**Discussion by ghost** [ansible-collections/ibm_zos_core#908] [Q&A] [answered]
I ran a task to copy a PDS member, as below:

  zos_copy:
    src: "{{ hlq_amb }}.PLEX{{ hlq_lib }}.PROCLIB.ANS({{ cics_modelo }})"
    dest: "{{ hlq_amb }}.PLEX{{ hlq_lib }}.PROCLIB.ANS({{ cics_novo }})"
    remote_src: true
    force: false (I also tried with tru)
    
And the result of that executions was this:

changed: [LABA] => {
    "changed": true,
    "dest": "LBS.PLEXLAB.PROCLIB.ANS(CICANSI)",
    "invocation": {
        "module_args": {
            "dest": "LBS.PLEXLAB.PROCLIB.ANS(CICANSI)",
            "force": false,
            "remote_src": true,
            "src": "LBS.PLEXLAB.PROCLIB.ANS(CIC9A01)"
        }
    },
    "is_binary": false,
    "src": "LBS.PLEXLAB.PROCLIB.ANS(CIC9A01)"
}

It turns out that the library was open by another user and the file was not created, but the task ended as if it had been.
As this module already has the force parameter, wouldn't it be the case to create another parameter to force the creation, even if the library is open (like parameter force in zos_lineinfile) or put some condition where the task fails?

**Comment by ddimatos** [comment, marked as answer]
Hello @fabiobncbb , you are correct in that the module does not support disp=shr. It should have been documented but given we are investing in this space we will forgo on documentation. In 1.7.0-beta.1 releasing in about a week; if you try this operation you will be presented an error rather than a false positive, see [586](https://github.com/ansible-collections/ibm_zos_core/issues/586). As for the use of force, we will be supporting disp=shr in 1.8.0-beta.1 planned for end of Q3, I'm uncertain if we will overload 'force' or introduce a new option , see [735](https://github.com/ansible-collections/ibm_zos_core/issues/735).

**Comment by ghost** [comment]
Thanks! It will be very useful for us!
